### PR TITLE
Improved Documentation and Added Patch

### LINKED ISSUE #296: Documentation: more friendly docs for beginners
**Describe your problem**

+ What is vineyard
+ and how vineyard works

**Additional context**

### PR DESCRIPTION
**Pull request by chaitravi-ce**
Signed-off-by: chaitravi-ce <chaitravi007@gmail.com>

## What do these changes do?

* Minor Changes in documentation which makes the documentation more user-friendly and easier to understand.
* Fixed some minor issues.
* Added a patch in troubleshooting section which resolves the libgrape-lite error while installing vineyard.

## Related issue number

Related to #296  but doesn't not fully fix it.

## Future Scope

* Adding reference to api documentation for easy accessibility to a certain function
* Adding more patches to FAQ section to describe vineyard

### CODE REVIEW

File: docs/notes/troubleshooting.rst [hunk 1]
```diff
@@ -54,3 +54,17 @@ could helps you when error occurs.
...
+
+  While attempting to install vineyard from source, user may face this error. A module 
+  named "Graph" in vineyard which is used for distributed data structure 
+  requires `libgrape-lite library <https://github.com/alibaba/libgrape-lite>`_ 
+  This issue can be resolved by:
+
+  + By installing the libgrape-lite library.
+  + Disabling the part by the following code:
+
+  .. code:: Python
```
Comment (sighingnow) [inline docs/notes/troubleshooting.rst:68]: The command is in bash, not in python.

```
.. code:: console

    cmake .. -DBUILD_VINEYARD_GRAPH=OFF
```

File: docs/notes/getting-started.rst [hunk 2]
```diff
@@ -138,6 +138,8 @@ is available for other clients:
      >>> value1
      ShareableList([b'a', 'bb', 1234, 56.78, False], name='o8000000119aa10c0')
 
+For more details, see :ref:`shared-memory`.
+
```
Comment (sighingnow) [inline docs/notes/getting-started.rst:142]: Nice fixes!

File: docs/notes/install.rst [hunk 3]
```diff
@@ -115,7 +117,7 @@ You can obtain the source code using ``git``:
 .. code:: console
 
     git clone https://github.com/v6d-io/v6d
-    cd libvineyard
+    cd v6d
```
Comment (sighingnow) [inline docs/notes/install.rst:120]: 👍

**Comment by sighingnow** [comment]
Once the comment above been addressed we can get this one landed.

**Review by andydiwenzhu — APPROVED** [review]
LGTM. @chaitravi-ce Thank you for the fix-up!

File: docs/notes/troubleshooting.rst (hunk 1 above)
Comment (chaitravi-ce) [reply docs/notes/troubleshooting.rst]: I have changed it to console now, please check

File: docs/notes/troubleshooting.rst [hunk 4]
```diff
@@ -54,3 +54,17 @@ could helps you when error occurs.
...
+  named "Graph" in vineyard which is used for distributed data structure 
+  requires `libgrape-lite library <https://github.com/alibaba/libgrape-lite>`_ .
+  This issue can be resolved by:
+
+  + By installing the libgrape-lite library.
+  + Disabling the part by the following code:
+
+  .. code:: console
+
+  >>> cmake .. -DBUILD_VINEYARD_GRAPH=OFF
```
Comment (sighingnow) [inline docs/notes/troubleshooting.rst:70]: Remove the `>>>` please, as it is not a python shell.

**Review by sighingnow — CHANGES_REQUESTED** [review]

**Review by sighingnow — APPROVED** [review]
LGTM.

**Comment by sighingnow** [comment]
Merged, thank you @chaitravi-ce!

I still leave #296 open for further improvements!

**Pull request merged**